app/login/page.js

```diff
@@ -0,0 +1,174 @@
+"use client"
+import React from 'react';
+import { useRouter } from 'next/navigation';
+import { useForm } from 'react-hook-form';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { z } from 'zod';
+import axios from 'axios';
+import { Button } from '@/components/ui/button';
+import { Input } from '@/components/ui/input';
+
+// Define Zod schema for form validation
+const schema = z.object({
+  email: z.string().email('Invalid email format').refine(data => {
+    // Custom validation function for email
+    const isValidEmail = /\S+@\S+\.(com|in)$/.test(data);
+    return isValidEmail || 'Invalid email format';
+  }),
+  password: z.string().min(3),
```

Comment:
3 characters are way to less for a password. Minimum password length should be at least six characters or more.

Reply:
fixed